package.json

```
@@ -97,6 +98,7 @@
     "rollup-plugin-polyfill-node": "^0.12.0",
     "rollup-plugin-typescript2": "^0.34.1",
     "rollup-plugin-visualizer": "^5.9.0",
+    "rollup-plugin-terser": "*",
```

I've just found rollup-plugin-terser is deprecated. Can we use [@rollup/plugin-terser](https://www.npmjs.com/package/@rollup/plugin-terser) instead and pin the version `^0.4.3`? 
```suggestion
    "@rollup/plugin-terser": "^0.4.3",
```